src/main/scala/is/hail/expr/Type.scala

```diff
@@ -699,23 +708,40 @@ abstract class TContainer extends Type {
     region.setBit(aoff + 4L, i.toL)
   }
 
-  def elementOffset(aoff: Long, length: Int, i: Int): Long =
+  def elementOffsetInRegion(aoff: Long, length: Int, i: Int): Long =
     aoff + elementsOffset(length) + i * elementByteSize
 
-  def elementOffset(region: MemoryBuffer, aoff: Long, i: Int): Long =
-    elementOffset(aoff, region.loadInt(aoff), i)
+  def elementOffsetInRegion(region: MemoryBuffer, aoff: Long, i: Int): Long =
+    elementOffsetInRegion(aoff, region.loadInt(aoff), i)
+
+  def elementOffsetInRegion(aoff: Code[Long], length: Code[Int], i: Code[Int]): Code[Long] =
```

Comment:
Shouldn't this just be `elementsOffset`?  The in region version takes a MemoryBuffer?

Reply:
ah shit yes, I screwed up the intellij rename apparently.